datastore/google/cloud/datastore/entity.py

```diff
@@ -24,9 +24,9 @@ class Entity(dict):
     An entity storing the actual instance of data.
 
     Each entity is officially represented with a
-    :class:`google.cloud.datastore.key.Key` class, however it is possible that
-    you might create an Entity with only a partial Key (that is, a Key
-    with a Kind, and possibly a parent, but without an ID).  In such a
+    :class:`~google.cloud.datastore.key.Key`, however it is possible that
+    you might create an entity with only a partial key (that is, a key
+    with a kind, and possibly a parent, but without an ID).  In such a
```

Comment:
These changes were some "editorializing" by me, happy to remove, though I think they are good changes.